src/test/java/racingcar/database/RacingCarDaoTest.java

```diff
@@ -0,0 +1,65 @@
+package racingcar.database;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.JdbcTest;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.test.context.jdbc.Sql;
+import racingcar.model.Car;
+
+@JdbcTest
+@Sql(scripts = {"classpath:data.sql"})
+class RacingCarDaoTest {
+    
+    private final RowMapper<Car> actorRowMapper = (resultSet, rowNum) -> {
+        return new Car(
+                resultSet.getString("name"),
+                resultSet.getInt("position")
+        );
+    };
+    @Autowired
+    private JdbcTemplate jdbcTemplate;
+    private RacingCarDao carDao;
```

Comment:
JdbcTemplate 처럼 Dao도 빈으로 주입받아 사용할 수 있어요 😄 

```suggestion
    @Autowired
    private RacingCarDao carDao;
```